app/src/main/java/ch/epfl/sweng/swenggolf/offer/ListAnswerAdapter.java

```diff
@@ -167,27 +163,35 @@ public void onClick(DialogInterface dialog, int which) {
         return builder.create();
     }
 
-    private Dialog stepBackDialog(Context context) {
-        AlertDialog.Builder builder = new AlertDialog.Builder(context);
+    private Dialog acceptAnswerDialog(Context context, final int pos) {
+        return createFavoriteDialog(context, pos,"Accept answer",
+                "Do you want to accept this answer?");
```

Comment:
Here (and below) you could put these strings in xml files so it is easier to translate the application later.
Tip : to do it fast, click on the string and use alt+enter. Then select "Extract string ressource".

Reply:
Well, I guess you'll read Elsa's code next time !

Reply:
Yes, thank you for fixing that, I was the one who wrote it that way first :)